hw/ip/otbn/dv/otbnsim/sim/sim.py

```diff
@@ -283,10 +283,8 @@ def _step_wiping(self, verbose: bool) -> StepRes:
 
         # If something bad happened asynchronously (because of an escalation),
         # we want to finish the secure wipe but accept no further commands.  To
-        # this end, set `STATUS` to Locked and turn this into a "wipe because
-        # something bad happended".
+        # this end, turn this into a "wipe because something bad happended".
         if self.state.pending_halt:
-            self.state.ext_regs.write('STATUS', Status.LOCKED, True)
```

Comment:
Did this behaviour change? I thought in the case of a fatal error we would always go to `LOCKED` state. Sorry if I'm missing something obvious :)